voice/bxml/verbs/speakSentence.md

```diff
@@ -69,7 +69,7 @@ Attributes:
  * `level`: (optional) defines the strength of emphasis to be applied, accepted values are: `strong`, `moderate` or `reduced`
 
 #### `<lang>`
-Specifies the natural language of the content.  For example, if you use an English American voice to speak a Spanish sentence, it will say the Spanish words with an American accent. 
+Specifies the natural language of the content. If you use a voice with an `en_US` locale, and a `es-MX` lang, it will sound like an American trying to speak Spanish. 
```

Comment:
```suggestion
Specifies the natural language of the content. If you use a voice with an `en_US` locale and an `es-MX` lang, it will sound like an English-speaking American attempting to speak Spanish.
```
This is the combination that sounded like someone who had no idea how Spanish works, right?

Reply:
well, it sounded like they were trying.  Without the lang it was nonsense